mu/virtual_environment.py

```diff
@@ -24,6 +25,40 @@
 logger = logging.getLogger(__name__)
 
 
+class SplashLogHandler(logging.NullHandler):

```

Comment:
I was wondering why only capture the venv stuff, can't we put this handler with the splash class in app.py and capture all the logs until the main window is drawn?

Reply:
I just imagined it was the venv stuff folks are most interested in. (More signal, less noise IYSWIM). Happy to change if need be.

Reply:
Yeah, it's definitely the part that would have the most interesting log messages. I don't feel strongly either way, and  I don't know if moving this to the GUI thread will have any other implications either (as it must have been hard to deal with some the bugs so far).